tst/teststandard/processes/slowwrite.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+
+use Time::HiRes;
+
+print $ARGV[1];
+select()->flush();
+Time::HiRes::sleep(1);
+
+print $ARGV[2];
```

Comment:
I am not so happy in now also adding perl scripts. Is there a strong reason for this? Can't the above also be done with an equally short bash/GAP/python script? All it seems to do is to print from argv, then flush and sleep. Shouldn't e.g. this do the job as well?
```sh
#!/bin/sh
echo $1
sleep 1
echo $2
```

Reply:
I mainly used perl, because there was already a perl script in the same directory

Reply:
That other script is perl because I needed to be able to ignore sigint and sigterm, and wasn't sure how to do that in any other language (well, other than C of course).

Reply:
However (should have done this before I started), now changed to bash. Ended up being a little trickier, as I need to control carefully where newlines appear, so I used printf instead of echo.